### PR TITLE
Add LottieView support on macOS using EpoxyCore 0.10.0

### PR DESCRIPTION
**Pull request by calda**
This PR:
 - Updates our EpoxyCore dependency to 0.10.0, which adds macOS support: https://github.com/airbnb/epoxy-ios/pull/143
 - Updates the SwiftUI `LottieView` to be available on macOS
 - Removes the tvOS and macOS example apps in favor of a single multiplatform SwiftUI example app that runs on all platforms. Eventually this will replace the iOS Example app as well.

I also ported some functionality from the UIKit example app to the SwiftUI example app, which now works on macOS and tvOS:

### CODE REVIEW

File: Example/Example/AnimationListView.swift [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            AnimationListView(directory: "\(directory)/\(subdirectoryURL.lastPathComponent)")
+          }
+        }
+      }
+    }.navigationTitle(directory)
+  }
+
+}
+
+extension AnimationListView {
```
Comment (calda) [inline Example/Example/AnimationListView.swift:46]: Everything in this extension was copy/pasted from the UIKit example app

File: Example/Example/AnimationListView.swift [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import Lottie
+import SwiftUI
+
+// MARK: - AnimationListView
+
+struct AnimationListView: View {
+
+  let directory: String
+
+  var body: some View {
```
Comment (calda) [inline Example/Example/AnimationListView.swift:13]: Nice and simple -- this was like 200 LOC plus the entire EpoxyCollectionView / EpoxyLayoutGroups dependency before in the UIKit example app

File: Example/Example/AnimationListView.swift [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    List {
+      ForEach(items, id: \.self) { item in
+        NavigationLink(value: item) {
```
Comment (calda) [inline Example/Example/AnimationListView.swift:16]: over here googling simple things like "swiftui how to push on navigation stack"

Comment (calda) [reply Example/Example/AnimationListView.swift:16]: I have now learned that there are 5-6 different APIs for doing this and I don't know the differences. Classic SwiftUI.

**Comment by calda** [comment]
ptal @brynbodayle

**Review by brynbodayle — APPROVED** [review]

**Pull request merged**